### ISSUE TITLE
OWASP Google Analytics

### ISSUE THREAD

**Issue by ThunderSon** [OWASP/CheatSheetSeries#386]
As per OWASP's request, we are going to add the google analytics to the cheatsheets website.
The following script needs to be appended on every page:
```javascript
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-4531126-15"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());
  gtag('config', 'UA-4531126-15');
</script>
```
@rejahrehim would you want me to assign this to you?

**Comment by rejahrehim** [comment]
Yes, Assign me

**Comment by rbsec** [comment]
@ThunderSon I presume you'll have access to the analytics data?

**Comment by ThunderSon** [comment]
@hblankenship I usually reach out to Harold and grab information when need be. Harold is there any dashboard we could access once it is set?

**Comment by ThunderSon** [comment]
Closed in #387
